package-lock.json

```diff
@@ -0,0 +1,3378 @@
+{
```

Comment:
Holy cow...one simple library....1000000 extra dependencies.  Curious how hard this would be to replicate by hand....

Reply:
OHHhhhh, I've been _not_ including the package-lock for the template